app/components/Dashboard/BestNewMusicTab/BestNewMusicMenu/BestNewMusicCard/index.js

```diff
@@ -1,40 +1,110 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Icon } from 'semantic-ui-react';
 
+import { removeQuotes, createLastFMLink } from '../../../../../utils';
+import { favoriteTrackShape } from '../../../../../constants/propTypes';
 import styles from './styles.scss';
 
+function toFavoriteTrack({ artist, title, thumbnail }) {
```

Comment:
best item has totally different structure. In order to add it to favorites we need to convert it.